polaris-service/src/main/java/org/apache/polaris/service/catalog/PolarisCatalogHandlerWrapper.java

```diff
@@ -834,7 +834,10 @@ public LoadTableResponse loadTableWithAccessDelegation(
             callContext.getPolarisCallContext(),
             catalogEntity,
             PolarisConfiguration.ALLOW_EXTERNAL_CATALOG_CREDENTIAL_VENDING)) {
-      throw new ForbiddenException("Access Delegation is not supported for this catalog");
+      throw new ForbiddenException(
+          "Access Delegation is not enabled for this catalog. Please consult applicable "
+              + "documentation for the catalog config property '%s' to enable this feature",
+          PolarisConfiguration.ALLOW_EXTERNAL_CATALOG_CREDENTIAL_VENDING.catalogConfig());
```

Comment:
The message is a little unclear; you can set the catalog property but you can also set the featureConfig. I also don't know if we need to say "please consult applicable documentation ..." rather than just saying e.g. "to enable this, set X or Y"

Reply:
My thinking on the wording was to avoid establishing a precedent of users starting to blindly set configs recommended by error messages. Admittedly it adds a bit of friction since this now makes it sound like there might be a more complicated series of steps to enable the feature rather than simply setting the config=true, but longer-term we should indeed probably standardize a way to point at relevant docs from error messages while also maybe making it easy for users to assess whether they care enough about the pros/cons of different config choices.

In any case, I totally agree we could have more clarity in the various config-related error messages, but I'm thinking we can do a bigger refactor to, for example, try to centralize error-message generation into `PolarisConfiuration.java` itself. But it'll take some refactoring of callsites.

Reply:
I see, it makes sense to avoid prompting users to blindly set a config. 

Ideally we can convey some nuance here, but I don't think the PolarisConfiguration descriptions are always adequate to do that. I'm concerned about pushing the error messages into there because some errors occur due to an interaction between configs or require other context.

In the future, we should look more closely at error messages but the currently-proposed change seems beneficial for the time being.